packages/compass-crud/src/stores/crud-store.ts

```diff
@@ -1235,37 +1244,20 @@ class CrudStoreImpl
       update,
     });
 
-    openToast('bulk-update-toast', {
-      title: '',
-      variant: 'progress',
-      dismissible: true,
-      timeout: null,
-      description: `${
-        this.state.bulkUpdate.affected || 0
-      } documents are being updated.`,
+    openBulkUpdateProgressToast({
+      affectedDocuments: this.state.bulkUpdate.affected || 0,
```

Comment:
If the count was `null` then we probably don't want to say that zero docs were updated or deleted, right? ie. we probably want what I did in this PR: https://github.com/mongodb-js/compass/pull/5144

Reply:
What would be the text on the successful toast? If we don't have a document count we can't show something like 'An indeterminate number of documents has been deleted'.

Maybe something like 'The delete/update operation finished successfully?'